src/Randomizer.Data/ViewModels/OptionsWindowTrackerProfiles.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using DynamicForms.Library.Core;
+using DynamicForms.Library.Core.Attributes;
+
+namespace Randomizer.Data.ViewModels;
+
+public class OptionsWindowTrackerProfiles : INotifyPropertyChanged
+{
+    private List<string> _availableProfiles = [];
+
+    [DynamicFormFieldText]
+    public string TrackerProfileDescription =>
+        "All selected profiles in the right column will be combined in the order from top to bottom. In case of overrides, such as Tracker item images, the top most will be used.";
```

Comment:
Is this label inaccurate now?

Reply:
I was thinking I was crazy because "I'm sure I changed the text!" Sure enough, I did. But only part of it. lol